docs/docs/app/data-source-types/mixpanel.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Setting up Mixpanel as a data source
+description: This document outlines the steps needed to add your Mixpanel database to GrowthBook.
+sidebar_label: Mixpanel
+slug: mixpanel
+---
+
```

Comment:
We should add a warning here, something like the following in a caution block, because we will deprecate Mixpanel as a datasource given they have deprecated the language we use to integrate directly with Mixpanel.

```
:::caution

Using Mixpanel as a direct datasource is not advised and has been placed in maintenance mode. This is because because Mixpanel has placed JQL, their query language, in [maintenance mode](https://developer.mixpanel.com/reference/jql-overview) themselves. Therefore, our integration is limited and may unexpectedly break as they wind down support for JQL.

In order to ensure long term compatibility with GrowthBook and to receive the full set of features available in GrowthBook, we suggest you set up an export to a data warehouse following [Mixpanel's documentation]([https://developer.mixpanel.com/reference/raw-data-export-api]) and then use that separate data warehouse (e.g. Snowflake, BigQuery) as your Data Source in GrowthBook.

:::
```

Reply:
We should add a similar warning here: docs/docs/app/event-trackers/mixpanel.mdx

steps 1 and 2 should be deprecated in favor of setting up another data source in mixpanel, and then following the instructions for connection that data source to GrowthBook using the appropriate data source page.

Reply:
I have added in the warnings.  

For this review I want it to be more of a refactor of what we had rather than adding/changing individual docs to make it easier to review.    Let's change steps 1 and 2 in this quide in a different PR once we allow for Bigquery or Snowflake in the Data Source Type options for the Mixpanel choice.

Reply:
Yep, that makes sense!